### PR TITLE
Add ksceKernelGetMutexInfo_089

### PR DESCRIPTION
**Pull request by al3xtjames**
https://wiki.henkaku.xyz/vita/SceKernelThreadMgr#sceKernelGetMutexInfo_089ForDriver

### CODE REVIEW

**Comment by CelesteBlue-dev** [comment]
Name is sceKernelGetMutexInfo_089ForDriver
so vitasdk name should be ksceKernelGetMutexInfo_089

**Comment by al3xtjames** [comment]
The wiki has the _089 suffix for other functions (ksceKernelGetMutexInfo, ksceKernelLockMutex, ksceKernelWaitThreadEnd, etc), but vitasdk omits them. Should this apply for ksceKernelGetMutexInfo?

**Comment by CelesteBlue-dev** [comment]
vitasdk was at first using guessed names, but when we know real name, write real name. Because later it might create duplicate name.

File: db.yml [hunk 1]
```diff
@@ -2246,6 +2246,7 @@ modules:
           ksceKernelExitDeleteThread: 0x1D17DECF
           ksceKernelExitThread: 0x0C8A38E1
           ksceKernelGetCallbackCount: 0x0892D8DF
+          ksceKernelGetMutexInfo_089: 0x69B78A12
```
Comment (d3m3vilurr) [inline db.yml]: can you remove `_089` postfix?

Comment (al3xtjames) [reply db.yml]: Done.

**Comment by CelesteBlue-dev** [comment]
me too I'm done

**Pull request merged**